src/diffusers/configuration_utils.py

```diff
@@ -358,8 +359,21 @@ def load_config(
                 "`self.config_name` is not defined. Note that one should not load a config from "
                 "`ConfigMixin`. Please make sure to define `config_name` in a class inheriting from `ConfigMixin`"
             )
-
-        if os.path.isfile(pretrained_model_name_or_path):
+        # Custom path for now
+        if dduf_entries:
+            if subfolder is not None:
+                raise ValueError("DDUF file only allow for 1 level of directory. Please check the DDUF structure")
+            # paths inside a DDUF file must always be "/"
+            config_file = (
+                cls.config_name
+                if pretrained_model_name_or_path == ""
+                else "/".join([pretrained_model_name_or_path, cls.config_name])
```

Comment:
```suggestion
                else os.path.join(pretrained_model_name_or_path, cls.config_name)
```

Reply:
in a DDUF file, one should always use `/` separator (same as on the Hub)